File: hackerrank/30 days of code/day 2, operators/operators.cs
// https://www.hackerrank.com/challenges/30-operators/problem?isFullScreen=true

using static System.Console;

class Solution
{
    public static void Main()
    {
        var mealCost = double.Parse(ReadLine());
        var tipPercent = int.Parse(ReadLine());
        var taxPercent = int.Parse(ReadLine());
        solve(mealCost, tipPercent, taxPercent);
    }

    private static void solve(double mealCost, int tipPercent, int taxPercent)
    {
        var totalCost = mealCost + mealCost * tipPercent / 100 + mealCost * taxPercent / 100;
        WriteLine("{0:0}", totalCost);
    }
}

File: hackerrank/30 days of code/day 2, operators/operators.c
// https://www.hackerrank.com/challenges/30-operators/problem?isFullScreen=true

#include <stdio.h>

void solve(const double meal_cost, const int tip_percent, const int tax_percent);

int main()
{
    double meal_cost;
    int tip_percent, tax_percent;
    scanf("%lf %d %d", &meal_cost, &tip_percent, &tax_percent);

    solve(meal_cost, tip_percent, tax_percent);

    return 0;
}

    void solve(const double meal_cost, const int tip_percent, const int tax_percent)
    {
        double total_cost = meal_cost + meal_cost * tip_percent / 100 + meal_cost * tax_percent / 100;
        printf("%.0lf", total_cost);
    }
